temporal-testing/src/test/java/io/temporal/testing/TestWorkflowEnvironmentSleepTest.java

```diff
@@ -51,16 +57,17 @@ protected void failed(Throwable e, Description description) {
   @WorkflowInterface
   public interface ExampleWorkflow {
     @WorkflowMethod
-    void execute();
+    String execute();
```

Comment:
I think you can use TestWorkflows.TestSignaledWorkflow here.

Reply:
Nvm, it's in a different package. Maybe just change the name to something more descriptive.